.travis.yml

```diff
@@ -29,9 +29,20 @@ install:
   - yarn bootstrap
 
 after_success:
-  - node_modules/.bin/codecov
+  - if ([ ! -z "$TRAVIS_TAG" ]); then
```

Comment:
i think you want https://yaml-multiline.info here with the "keep newlines" syntax (just `|`)